lib-include/difftest.h

```diff
@@ -31,13 +34,16 @@ enum { DIFFTEST_TO_DUT, DIFFTEST_TO_REF };
 #elif defined(__ISA_riscv32__)
 # define DIFFTEST_REG_SIZE (sizeof(uint32_t) * 33) // GRPs + pc
 #elif defined(__ISA_riscv64__)
-#ifdef RV64_FULL_DIFF 
+#if defined RV64_FULL_DIFF 
 #ifdef CONFIG_RVV 
 #define DIFFTEST_REG_SIZE (sizeof(uint64_t) * (32 + 32 + 1 + 6 + 11 + 1 + 64 + 7))
 #else
 #define DIFFTEST_REG_SIZE (sizeof(uint64_t) * (32 + 32 + 1 + 6 + 11 + 1))
 #endif
 // GRPs + FPRs + pc + [m|s][status|cause|epc] + other necessary CSRs + mode
+#elif defined RV64_FULL_DIFF_H
+#define DIFFTEST_REG_SIZE (sizeof(uint64_t) * (32 + 32 + 1 + 6 + 11 + 1 + 1 + 16))
+// GRPs + FPRs + pc + [m|s][status|cause|epc] + other necessary CSRs + mode + v-mode + HCSRS 
```

Comment:
The REG_SIZE "framework" is too ugly now. May be we can change it into

# ifdef **
# define RVV_EXT_REG_SIZE ***
# else
***
# define SIZE (BASE_SIZE + RVV_EXT_REG_SIZE + H_EXT_REG_SIZE + ...)

Reply:
I will change it.